.markdownlint.json

```diff
@@ -6,6 +6,23 @@
   "MD013": false,
   "MD025": { "front_matter_title": "" },
   "MD026": { "punctuation": ".,;:!" },
-  "MD033": false,
+  "MD033": {
+    "allowed_elements": [
+      "a",
+      "abbr",
+      "b",
+      "code",
+      "div",
+      "em",
+      "figcaption",
+      "figure",
+      "i",
+      "img",
+      "kbd",
+      "pre",
+      "sub",
+      "sup"
```

Comment:
I wonder how this passes even though `iframe` is missing from the list (and we use it for YouTube embeds in a few places). Any ideas?

Reply:
Could be a bug or part of the regex. I did notice the iframe would get flagged if I removed "div" since that was the element nesting them AFAIK

Reply:
Interesting. I guess it only covers "outer" elements.

Reply:
Some of the inner ones seem to be flagged like "em", but maybe it's just the direct decendants of an outer element. Or just a regex bug :)